packages/arkor-local/python/mlx/dataset_prep.py

```
@@ -0,0 +1,254 @@
+    return train, valid
+
+
+def _load_blob(source: dict, log):
+    url = source["url"]
+    token = source.get("token")
+    log(f"[arkor] downloading blob dataset from {url}")
+    request = urllib.request.Request(url)
+    if token:
+        request.add_header("Authorization", f"Bearer {token}")
+    try:
+        with urllib.request.urlopen(request) as response:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)